views/pages/index.ejs

```diff
@@ -1,12 +1,29 @@
-<%- include("../partials/header.ejs", {title: "Main Page"}) %>
 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
```

Comment:
Instead of "Home page", this title might be better as a more descriptive "Naz and Tomi's TV Show Episodes".  The html title is what gets shown in the browser tab or as the default name of the  bookmark when you bookmark your site in the browser.  "home page" wouldn't really tell anyone what site the tab /bookmark is on.